src/mcts/search.cc

```diff
@@ -401,6 +404,7 @@ EdgeAndNode Search::GetBestChildNoTemperature(Node* parent) const {
 // Returns a child chosen according to weighted-by-temperature visit count.
 EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
                                                 float temperature) const {
+  assert(parent->GetN() > 1); // Require at least one visited child
```

Comment:
Maybe `parent->GetChildrenVisits() >= 1`?  It's the same, but function name matches comment better.
Also comment in that case is not needed (as it just repeats what statement says), but if you want to keep the comment, then period at the end of sentence. :)

Reply:
In theory, yes, but in practice, per my comment on Discord, `GetChildrenVisits` returns `1` when `n_` is 0, so is not suitable for this purpose.

I suspect there are many other such uses of `GetN` scattered about the codebase that can be so changed, only after we fix GetChildrenVisits, which I think is inappropriate for *this* PR. (There are several other minor/inconsequential blemishes I've been thinking about touching up, and fixing GCV is one of many things I'll do all at once.)

Reply:
Yes, that GetChildrenVisits() returning 1 looks like a bug, so maybe also fix it in this PR?

All places calling that function seems to expect that it returns 0 sometimes.